client/rest-high-level/src/main/java/org/elasticsearch/client/eql/EqlSearchRequest.java

```diff
@@ -158,8 +158,8 @@ public int size() {
 
     public EqlSearchRequest size(int size) {
         this.size = size;
-        if (fetchSize <= 0) {
-            throw new IllegalArgumentException("size must be greater than 0");
+        if (fetchSize < 0) {
```

Comment:
Shouldn't this be `size` and not `fetchSize`?

Reply:
🤦